river/InputManager.zig

```diff
@@ -95,7 +95,7 @@ relative_pointer_manager: *wlr.RelativePointerManagerV1,
 virtual_pointer_manager: *wlr.VirtualPointerManagerV1,
 virtual_keyboard_manager: *wlr.VirtualKeyboardManagerV1,
 
-input_configs: std.SinglyLinkedList(InputConfig) = .{},
+input_configs: std.ArrayList(InputConfig) = undefined,
```

Comment:
We should not use a default value here, as omitting `input_configs` when initializing `InputManager` would be a bug.